controllers/cards.js

```diff
@@ -1,69 +1,56 @@
 const Card = require('../models/cards');
+const ServerError = require('../errors/server-err');
+const NotFoundError = require('../errors/not-found-err');
+const NotEnoughRightsError = require('../errors/not-enough-rights-err');
 
-module.exports.getCards = (req, res) => {
+module.exports.getCards = (req, res, next) => {
   Card.find({})
     .then((cards) => res.send(cards))
-    .catch((error) => res.status(500)
-      .send({ message: error.message }));
+    .catch(() => next(new ServerError('Произошла неизвестная ошибка')));
```

Comment:
Можно лучше: упростить обработку _Internal Server Error_ централизованным обработчиком ошибок.
Сейчас практически все контроллеры при обработке исключений создают экземпляр `ServerError` при возникновении ошибки. Естветсвенно, если эта ошибка связана с ошибкой на сервере. Сам код не идентичен, потому что текст меняется. Местами `Произошла неизвестная ошибка` или `Неверный запрос`.

Можно чуть упростить код и избавиться от необходимости постоянно создавать экземпляр этой ошибки в каждом контроллере. Например, можно определить `HTTP` код и нужный текст сообщения непосредственно в обработчике ошибок. Тогда `.catch()` блок в большинстве случаев будет выглядеть так:
```suggestion
    .catch(next);
```

Ошибка просто идёт дальше и подхватывается обработчиком:
```js
// eslint-disable-next-line no-unused-vars
app.use((err, req, res, next) => {
  const statusCode = err.statusCode || 500;
  const message = statusCode === 500 ? 'Произошла ошибка на сервере' : err.message;
  
  res.status(statusCode).send({ message });
});
```

Который понимает, что если `statusCode` нет у ошибки, то это скорее всего наш базовый случай. Поэтому фиксируем текст сообщения и отправляем клиенту.